modules/onmyoji/cogs/shard.py

```diff
@@ -465,21 +479,21 @@ def check_trading_status(self, user):
                 temp = self.shard_trading_db[str(ctx.author.id)]['status']
             except KeyError:
                 self.shard_trading_db[str(ctx.author.id)]['status'] = True
-                self.shard_file_writeout();
+                self.shard_file_writeout_all();
                 return await ctx.send(f"{ctx.author.mention} is now available to be searched for trading.")
             trading_status = 'available' if  self.check_trading_status(ctx.author.id) else 'unavailable'
             return await ctx.send(f"{ctx.author.mention} is currently {trading_status} to be searched for trading.")
 
     @shard_status.command(name="on")
     async def shard_status_set_on(self, ctx):
         self.shard_trading_db[str(ctx.author.id)]['status'] = True
-        self.shard_file_writeout();
+        self.shard_file_writeout_all();
```

Comment:
semicolons?

Reply:
I have no idea why those are there, I guess I sneezed and Java came out.